app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -271,6 +271,21 @@ input.search-bar {
   }
 }
 
+.btn-link {
+  background-color: transparent;
+  border: 0;
+  color: #0d6efd;
+
+  &:hover {
+    color: #0a58ca !important;
```

Comment:
I'm not sure we should harcode this. If your PrimaryColor is red, this would look very weird

Reply:
![image](https://user-images.githubusercontent.com/43917914/194382763-1be3eb85-0f27-415e-b2e1-9c061025f20e.png)

Reply:
the "Sign In" is a button since the OPENID connect is a form. I created the "link" variant so it looks like an <a> link.